ross/rotor_assembly.py

```diff
@@ -3821,10 +3821,21 @@ def rotor_example_6dof():
     >>> import ross as rs
     >>> import numpy as np
     >>> rotor6 = rs.rotor_assembly.rotor_example_6dof()
-    >>> camp6 = rotor6.run_campbell(np.linspace(0,400,101),frequencies=18)
 
-    # plotting Campbell Diagram
+    Plotting rotor model
```

Comment:
Hi @verg1lio ! Going [here](https://github.com/ross-rotordynamics/ross/pull/826/checks?check_run_id=3086398825) you can see what tests are failing in the PR.
Notice that we also test the examples in the docstrings. We do this to be sure that all code provided in the examples are always working.
The rules for this part of the docstrings are: 
- Everything with '>>>' is code that will be executed;
- Everything that does **not** have '>>>' is output, and this value will be compared with the code output for the test to pass;
- All comments should have '#' to avoid execution.
Here the text 'Plotting rotor model' does not have the '#', since this is not valid code the test will fail.